ipaclient/install/client.py

```diff
@@ -3274,6 +3270,23 @@ def _install(options):
         configure_nisdomain(
             options=options, domain=cli_domain, statestore=statestore)
 
+    # Configure the final krb5.conf
+    if not options.on_master:
+        fstore.backup_file(paths.KRB5_CONF)
+        configure_krb5_conf(
+            cli_realm=cli_realm,
+            cli_domain=cli_domain,
+            cli_server=cli_server,
+            cli_kdc=cli_kdc,
+            dnsok=dnsok,
+            filename=paths.KRB5_CONF,
+            client_domain=client_domain,
+            client_hostname=hostname,
+            configure_sssd=options.sssd,
+            force=options.force)
+
+        logger.info("Configured /etc/krb5.conf for IPA realm %s", cli_realm)
+
```

Comment:
Nitpick, but at this point (or at the end of the cleanup wrapper) we could call `del os.environ['KRB5_CONFIG']` and rely on the normal /etc/krb5.conf file.

Reply:
Sure I can add that. I don't think it will make much of a difference now but it's good future-proofing.